build-deploy-container_nmma.yml

```diff
@@ -0,0 +1,44 @@
+name: CI to Docker Hub
+
+on:
+  push:
+    branches: [ main ]
+env:
+  REGISTRY_IMAGE: nmma/nmma
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Check Out Repo
+        uses: actions/checkout@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY_IMAGE }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      
+    - name: Build Docker image
```

Comment:
If you open this file in Github editor the following error pops up:
`Bad indentation of a mapping entry`